pandas/_libs/tslibs/offsets.pyx

```diff
@@ -745,26 +822,96 @@ cdef class BaseOffset:
     def is_anchored(self) -> bool:
         # TODO: Does this make sense for the general case?  It would help
         # if there were a canonical docstring for what is_anchored means.
+        """
+        Return boolean whether the frequency is a unit frequency.
```

Comment:
I don't understand this - but I don't use these functions frequently.

Reply:
Added a note here that `unit` means `(n=1)`